ops/internal/manage/testdata/superchain/configs/sepolia/superchain.toml

```
@@ -1,7 +1,7 @@
 name = "Sepolia"
 protocol_versions_addr = "0x79ADD5713B383DAa0a138d3C4780C7A1804a8090"
 superchain_config_addr = "0xC2Be75506d5724086DEB7245bd260Cc9753911Be"
-op_contracts_manager_proxy_addr = "0xF564eEA7960EA244bfEbCBbB17858748606147bf"
+op_contracts_manager_addr = "0xF564eEA7960EA244bfEbCBbB17858748606147bf"
```

Hm, this format is outdated too, and it should be something like this now:
```suggestion
[opcm]
"v1.8.0" = "0x123..."
"v2.0.0" = "0x456..."
```

---

actually this will need to be keyed by network name also
```toml
[opcm]
# Key refers to the L1 contracts release
"v1.8.0" = {mainnet = "0x123...", sepolia = "0xabc..." }
"v2.0.0" = {mainnet = "0x456...", sepolia = "0xdef..." }
```